patch/secure_boot_kernel_config.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# Note: this script was created because there is a problem when changing the kernel config
+# values that requires in the Secure Boot feature when using patch/kconfig-inclusions (sonic flow to modify kernel flags).
+# So, when this problem will be resolved, this script should be removed and used the kconfig-inclusions.
```

Comment:
Where is that problem documented? Please open an issue with the details.

Reply:
https://github.com/sonic-net/sonic-linux-kernel/issues/299